tdrs-backend/tdpservice/parsers/validators/category2.py

```diff
@@ -0,0 +1,191 @@
+"""Overloaded base validators and custom validators for category 2 validation (field validation)."""
+
+from tdpservice.parsers.util import clean_options_string
+from . import base
+from .util import ValidationErrorArgs, validator, make_validator
+
+
+def format_error_context(eargs: ValidationErrorArgs):
+    """Format the error message for consistency across cat2 validators."""
+    return f'{eargs.row_schema.record_type} Item {eargs.item_num} ({eargs.friendly_name}):'
+
+
+@validator(base.isEqual)
```

Comment:
This is a thought, and doesn't necessarily need any research/implementation... But what if instead of decorating the validator we decorate the error message? Maybe that doesn't make sense, just an off the cuff thought.

Reply:
Just want to clarify that my thought process on this is that the Decorator pattern is used to add/change data to a class/function . Since the error message (and not the validator itself) is changing it seems like we should decorate the validator with the error message as opposed to the other way around.

Reply:
error strings require args from the validator function as well as eargs from the error generator